### DISCUSSION TITLE
Sub-dependencies?

### DISCUSSION THREAD

**Discussion by bortels** [pkgxdev/pkgx#261] [Help]
So - maybe I'm missing it, but what is the recommended mechanism for sub-dependencies? 

For example - I have a python script, and it does "import requests", as you do.  

Obviously - that needs to be pip-installed before the script executes. How does one do this, or is this still TBI? 

it would be nice if it recognized a requirements.txt or could pull those from the README.md, or even looked for "import" statements in the python code. 

Ditto other languages - I think the only one this maybe isn't a problem in is deno (w00t go deno). 

PS. Note it's "README.md" - a plain old 1980's README (ie. not markdown) is ignored. The README.md refers to it randomly by both names; maybe if no README.md is found it might be nice to try README as a last resort.

**Comment by mxcl** [comment]
Stay tuned! We have plans to add control of eg. `pip` to the front matter YAML.

**Comment by mxcl** [comment]
> The README.md refers to it randomly by both names; maybe if no README.md is found it might be nice to try README as a last resort.

Fair point. I'll noodle it. The other option is `package.json` or the upcoming `package.yaml` from this discussion: https://github.com/teaxyz/cli/issues/201
